cfgov/templates/wagtailadmin/js/table-block.js

```diff
@@ -33,7 +33,8 @@
                 HEIGHT: 1
             },
 
-            $colWidthSelect: {}
+            $colWidthSelect: {},
+            $sortableSelect: {}
```

Comment:
I thought we didn't add trailing commas to `js` files, sort of as a carry over from `json` formatting.